site/src/components/Markdown/Markdown.tsx

```diff
@@ -113,7 +113,66 @@ export const Markdown: FC<MarkdownProps> = (props) => {
   );
 };
 
+interface MarkdownProps {
+  /**
+   * The Markdown text to parse and render
+   */
+  children: string;
+
+  className?: string;
+
+  /**
+   * Can override the behavior of the generated elements
+   */
+  components?: Options["components"];
+}
+
+/**
+ * Supports a strict subset of Markdown that bahaves well as inline/confined content.
+ */
+export const InlineMarkdown: FC<MarkdownProps> = (props) => {
+  const { children, className, components = {} } = props;
+
+  return (
+    <ReactMarkdown
+      css={markdownStyles}
+      className={className}
+      allowedElements={["p", "em", "strong", "a", "pre", "code"]}
```

Comment:
I'm a little confused here – I think my CSS brain is throwing me for a loop

What counts as inline elements for this component? Is it just `<p>` elements and any text-based content that could go inside them?

Reply:
Also, since there aren't components defined for all of the allowed elements, is there a default component that gets used, or do the elements just render as plain text?

Reply:
Okay, cool. That makes sense